.travis_support/environments/dpl.yml

```diff
@@ -1,6 +1,7 @@
 name: dask_image_dpl_env
 
 channels:
+  - defaults
```

Comment:
Is this still needed?

Reply:
As it turns out, yes.

I watched the builds pass, then tried removing the defaults channel and saw it fail:
* https://travis-ci.org/GenevieveBuckley/dask-image/jobs/558679819
* https://travis-ci.org/GenevieveBuckley/dask-image/jobs/558679820

This was the relevant part:
```
+++/Users/travis/miniconda/bin/conda env create -p /var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/tmpmiq6qeat/dpl -f .travis_support/environments/dpl.yml
Solving environment: done
==> WARNING: A newer version of conda exists. <==
  current version: 4.5.12
  latest version: 4.7.5
Please update conda by running
    $ conda update -n base -c defaults conda
Downloading and Extracting Packages
cffi-1.12.3          | 210 KB    | : 100% 1.0/1 [00:00<00:00,  6.08it/s]               
certifi-2019.6.16    | 147 KB    | : 100% 1.0/1 [00:00<00:00, 15.88it/s]
pyopenssl-19.0.0     | 81 KB     | : 100% 1.0/1 [00:00<00:00, 14.39it/s]
urllib3-1.25.3       | 187 KB    | : 100% 1.0/1 [00:00<00:00,  6.27it/s]
docopt-0.6.2         | 14 KB     | : 100% 1.0/1 [00:00<00:00, 18.66it/s]
python-3.6.7         | 20.6 MB   | : 100% 1.0/1 [00:04<00:00,  4.80s/it]               
six-1.12.0           | 22 KB     | : 100% 1.0/1 [00:00<00:00, 18.22it/s]
pycparser-2.19       | 173 KB    | : 100% 1.0/1 [00:00<00:00,  7.13it/s]
chardet-3.0.4        | 190 KB    | : 100% 1.0/1 [00:00<00:00,  7.62it/s]               
idna-2.8             | 132 KB    | : 100% 1.0/1 [00:00<00:00, 13.31it/s]
coverage-4.5.1       | 211 KB    | : 100% 1.0/1 [00:00<00:00,  7.21it/s]
pip-18.0             | 1.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.28it/s]               
pysocks-1.7.0        | 26 KB     | : 100% 1.0/1 [00:00<00:00, 16.63it/s]
asn1crypto-0.24.0    | 154 KB    | : 100% 1.0/1 [00:00<00:00,  8.48it/s]
yaml-0.1.7           | 79 KB     | : 100% 1.0/1 [00:00<00:00, 18.79it/s]
cryptography-2.7     | 566 KB    | : 100% 1.0/1 [00:00<00:00,  2.88it/s]               
setuptools-41.0.1    | 620 KB    | : 100% 1.0/1 [00:00<00:00,  3.67it/s]               
wheel-0.31.1         | 62 KB     | : 100% 1.0/1 [00:00<00:00, 15.64it/s]
coveralls-1.3.0      | 12 KB     | : 100% 1.0/1 [00:00<00:00, 27.28it/s]
requests-2.22.0      | 84 KB     | : 100% 1.0/1 [00:00<00:00, 13.92it/s]
pyyaml-5.1.1         | 166 KB    | : 100% 1.0/1 [00:00<00:00,  9.40it/s]
Preparing transaction: done
Verifying transaction: done
Executing transaction: failed
ERROR conda.core.link:_execute(507): An error occurred while installing package 'conda-forge::docopt-0.6.2-py_1'.
FileNotFoundError(2, "No such file or directory: '/var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/tmpmiq6qeat/dpl/bin/python3.6'")
Attempting to roll back.
Rolling back transaction: done
FileNotFoundError(2, "No such file or directory: '/var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/tmpmiq6qeat/dpl/bin/python3.6'")
```